include/cantera/transport/HighPressureGasTransport.h

```diff
@@ -66,26 +112,550 @@ class HighPressureGasTransport : public MultiTransport
+     */
+    double Vcrit_i(size_t i);
+
+    /**
+     * @brief  Returns the estimate of the critical compressibility that is given
+     * from the thermo object for species i.
+     *
+     * This method sets the species composition vector to unity for
+     * species i and zero for all other species, and then queries the
+     * thermo object for the critical compressibility. It then resets the
+     * composition vector to the original state.
+     *
+     * @param i  Species index
+     * @return double
+     */
+    double Zcrit_i(size_t i);
+
+
+    /**
+     * Returns the composition-dependent values of the parameters needed for
+     * the Lucas viscosity model.
+     *
+     * The equations for the mixing rules defined on page 9.23 for the Lucas method's
+     * composition dependent parameters. The primary mixing rules are defined below,
+     * and the reduced properties are just the properties divided by the pseudo-critical
+     * mixture properties defined below.
+     *
+     * @f[
+     *  T_{\t{c,m}} = \sum_i y_i T_{\t{c,i}}
+     * @f]
+     *
+     * @f[
+     *  P_{\t{c,m}} = R T_{\t{c,m}} \frac{\sum_i y_i Z_{\t{c,i}}}{\sum_i y_i V_{\t{c,i}}}
+     * @f]
+     *
+     * @f[
+     *  M_m = \sum y_i M_i
+     * @f]
+     *
+     * @f[
+     *  F_{P,m}^{\t{o}} = \sum y_i F_{P,i}^{\t{o}}
+     * @f]
+     *
+     * @f[
+     *   F_{Q,m}^{\t{o}} = \left ( \sum y_i F_{Q,i}^{\t{o}} \right ) A
+     * @f]
+     *
+     * @f[
+     *   A = 1 - 0.01 \left ( \frac{M_H}{M_L} \right )^{0.87}
+     * @f]
+     *
+     * For $\frac{M_H}{M_L} > 9$ and $ 0.05 < y_H < 0.7$, otherwise A = 1. In the
+     * above equation, $M_H$ and $M_L$ are the molecular weights of the heaviest
+     * and lightest components in the mixture, and $y_H$ is the mole fraction of
+     * the heaviest component.
+     *
+     * While it isn't returned as a parameter, the species-specific reduced dipole
+     * moment is used to compute the mixture polarity correction factor. It is
+     * defined as:
+     *
+     * @f[
+     *   \mu_r = 52.46 \frac{\mu^2 P_{\t{c,i}}}{T_{\t{c,i}}
+     * @f]
+     *
+     */
+    void compute_mixture_parameters(LucasMixtureParameters& params);
+
+    /**
+     * Returns the non-dimensional low-pressure mixture viscosity in using the Lucas method.
+     *
+     * Defined by equation 9-4.16.
+     *
+     * @f[
+     *   \eta \xi = [0.807 T_r^{0.618} - 0.357 e^{-0.449 T_r} + 0.340e^{-4.058 T_r} + 0.018] F_P^{\t{o}} F_Q^{\t{o}}
+     * @f]
+     *
+     * This function is structured such that it can be used for pure species or mixtures, with the
+     * only difference being the values that are passed to the function (pure values versus mixture values).
+     *
+     * @param Tr Reduced temperature [unitless]
+     * @param FP Polarity correction factor [unitless]
+     * @param FQ Quantum correction factor [unitless]
+     * @return double
+     */
+    double low_pressure_nondimensional_viscosity(double Tr, double FP, double FQ);
+
```

Comment:
C++ member function names should use camel case names:
```suggestion
    double lowPressureNondimensionalViscosity(double Tr, double FP, double FQ);

```